deployment/build-open-source-dist.sh

```diff
@@ -0,0 +1,74 @@
+#!/bin/bash
+#
+# This assumes all of the OS-level configuration has been completed and git repo has already been cloned
+#
+# This script should be run from the repo's deployment directory
+# cd deployment
+# ./build-s3-dist.sh solution-name
```

Comment:
The name of this file is build-open-source-dist.sh should we change this comment to reflect the same?

Reply:
good spot, updated.